File: inventory_Borja.c
#include <stdio.h>

#define SIZE 3  

void add_to_stock(int stock[], int *size) {
    stock[*size] = 75;  
    (*size)++;  
    printf("Inside function (stock): [");
    for (int i = 0; i < *size; i++) {
        printf("%d", stock[i]);
        if (i < *size - 1) printf(", ");
    }
    printf("]\n");
}

void update_price(float price) {
    float new_price = price + (price * 0.10f);
    printf("Inside function (price): %.1f\n", new_price);
}

int main() {
    int inventory[10] = {100, 200, 150};  
    int size = SIZE;

    add_to_stock(inventory, &size);
    printf("Outside function (stock): [");
    for (int i = 0; i < size; i++) {
        printf("%d", inventory[i]);
        if (i < size - 1) printf(", ");
    }
    printf("]\n\n");

    float base_price = 250.0f;
    update_price(base_price);
    printf("Outside function (price): %.1f\n", base_price);

    return 0;
}

File: inventory_Borja.py
def add_to_stock(stock_list):
    stock_list.append(75)
    print("Inside function (stock):", stock_list)

def update_price(price):
    new_price = price + (price * 0.10)
    print("Inside function (price):", new_price)

inventory = [100, 200, 150]
add_to_stock(inventory)
print("Outside function (stock):", inventory)

print()  

base_price = 250.0
update_price(base_price)
print("Outside function (price):", base_price)
